src/model_repository_manager.h

```diff
@@ -422,14 +455,24 @@ class ModelRepositoryManager {
   Status GetModelInfo(
       const ModelIdentifier& model_id, ModelInfo** model_info) const;
 
+  /// Get the model info for a named model, from the provided model infos.
+  /// \param model_infos The model infos where the model is to be lookup.
+  /// \param name The model name.
+  /// \param model_info Returns the model information.
+  /// \return OK if found, NOT_FOUND otherwise.
+  static Status GetModelInfo(
```

Comment:
It does not, it belongs to the ModelInfo map held by the manager class. I think it would be better to refactor the map to its own class rather than depending on `std::unordered_map`.

Reply:
Update commit: Group ModelInfoMap functions